src/Symfony/Bundle/TwigBundle/Resources/views/Exception/trace.html.twig

```diff
@@ -1,3 +1,13 @@
+<div>
+{% spaceless %}
+    <span class="toggle-icon">
+        <a href="#" onclick="toggle('data-{{ prefix ~ '-' ~ i }}'); switchIcons('icon-{{ prefix ~ '-' ~ i }}-open', 'icon-{{ prefix ~ '-' ~ i }}-close'); return false;">
```

Comment:
@jrobeson I don't know if I understand. Also, this is just a movement of this code - we're already using the onclick like this (search for `toggle(` on this page - you'll see the part is was moved from)